airgun/entities/cloud_insights.py

```diff
@@ -49,6 +60,93 @@ def update(self, values):
+
+    def search(self, value):
+        """Search for 'query' and return matched hostnames/recommendations.
+
+        :param value: text to filter (default: no filter)
+        """
+        view = self.navigate_to(self, 'All')
+        time.sleep(5)
+        view.clear_button.click()
+        view.search_field.fill(value)
+        time.sleep(5)
+        return view.table.read()
+
+    def remediate_affected_systems(self, recommendation_name, hostname):
+        """Open Affected systems, filter by hostname, select it, and click Remediate.
+
+        Returns the details view contents after remediation click.
+        """
+        # Use navigator to open the Affected Systems details view
+        view = self.navigate_to(self, 'Affected Systems', recommendation_name=recommendation_name)
+        view.search_field.wait_displayed()
+        # Filter by hostname and apply recommendation
+        view.search_field.fill(hostname)
+        wait_for(lambda: view.table.row(name=hostname), handle_exception=True, timeout=20)
+        time.sleep(15)
+        view.table[0][0].widget.click()
+        view.remediate.click()
+        self.browser.plugin.ensure_page_safe(timeout='30s')
+        modal = RemediateSummary(self.browser)
+        if modal.is_displayed:
+            modal.remediate.click()
+        view = JobInvocationStatusView(view.browser)
+        view.wait_for_result()
+        return view.read()
+
+    def bulk_remediate_affected_systems(self, recommendation_name):
+        """Open Affected systems, bulk select affected systems, and click Remediate.
+
+        Returns the details view contents after remediation click.
+        """
+        # Use navigator to open the Affected Systems details view
+        view = self.navigate_to(self, 'Affected Systems', recommendation_name=recommendation_name)
+        # view.search_field.wait_displayed()
+        wait_for(lambda: view.table.row(), handle_exception=True, timeout=20)
+        time.sleep(5)
+        view.bulk_select.select_all()
+        view.remediate.click()
+        self.browser.plugin.ensure_page_safe(timeout='30s')
+        modal = RemediateSummary(self.browser)
+        if modal.is_displayed:
+            modal.remediate.click()
+        view = JobInvocationStatusView(view.browser)
+        view.wait_for_result()
+        return view.read()
+
+    def read(self, widget_names=None):
+        """Read all values."""
+        view = self.navigate_to(self, 'All')
+        self.browser.plugin.ensure_page_safe(timeout='10s')
+        view.wait_displayed()
+        return view.read(widget_names=widget_names)
+
+
+@navigator.register(RecommendationsTabEntity, 'Affected Systems')
+class NavigateToAffectedSystems(NavigateStep):
+    """Navigate from Recommendations tab to the Affected Systems details view."""
+
+    VIEW = RecommendationsDetailsView
+
+    def prerequisite(self, *args, **kwargs):
+        # Ensure we are on the Recommendations tab first
+        return self.navigate_to(self.obj, 'All')
```

Comment:
```suggestion
        return self.navigate_to(self.obj, 'Recommendations')
```
Change linked to [this review comment](https://github.com/SatelliteQE/airgun/pull/1983/files#r2374593279).